README.md

```diff
@@ -4,107 +4,188 @@ This repository contains the Nervos BOLOS application for the Ledger Nano S and
 
 This application has been developed against our forks of [CKB-CLI](https://github.com/obsidiansystems/ckb-cli) and [CKB](https://github.com/obsidiansystems/ckb). Most instructions assume you have the [Nix](https://nixos.org/nix/) Package Manager, which you can install on any Linux distribution or MacOS. Application and wallet developers who would like to support Ledger can do so with [LedgerJS](https://github.com/obsidiansystems/ledgerjs/tree/nervos).
 
-# Installation on the Nano S #
+# Prepare machine to talk to ledger
 
-To install from the source directory, ensure that you have nix installed, run
+On Linux, the "udev" rules must be set up to allow your user to communicate with the ledger device.
 
-```
-$ ./nix/install.sh s
+## NixOS
+
+On NixOS, one can easily do this with by adding the following to configuration.nix:
+``` nix
+{
+  # ...
+  hardware.ledger.enable = true;
+  # ...
+}
 ```
 
-and accept the prompts on your ledger device.
+## Non-NixOS Distros
```

Comment:
Somewhere in this section I think it makes sense to reference https://github.com/LedgerHQ/udev-rules and https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues in particular for users who may have issues and would benefit from Ledger's documentation on this